### PR TITLE
Fix state name

### PR DESCRIPTION
**Pull request by wiedehoeft**
I had to update dependencies and change the test structure.
Hope the result now, is same as expected before. The pattern returning a promise at the end of a test is something I have never seen.
If it is not the expected result, the last commit should be enough, but the tests are not executable then.

### CODE REVIEW

File: test/tests.spec.js [hunk 1]
```diff
@@ -5,119 +5,125 @@ describe('angular-promise-extras', function () {
...
-      .then(done, done.fail)
+        .then(done, done.fail)
 
       deferreds[0].resolve(1)
       deferreds[1].reject(2)
       deferreds[2].resolve(3)
 
       $rootScope.$apply()
-      return testProm
+      done()
```
Comment (insidewhy) [inline test/tests.spec.js:66]: This actually stops the tests from testing. Return the promise as before.

**Review by insidewhy — CHANGES_REQUESTED** [review]
Also when contributing to a new project you shouldn't reformat all the code.

**Comment by insidewhy** [comment]
> The pattern returning a promise at the end of a test is something I have never seen.

This is the usual now, the done callback is harder to use. By changing the tests to call done at the end as you did, you are making the tests end before the promises resolve which stops them from asserting.

**Comment by wiedehoeft** [comment]
Than a new version with changed name only. Tests will not be executable.

**Comment by wiedehoeft** [comment]
> Also when contributing to a new project you shouldn't reformat all the code.

Then a file should be attached how the code should be formatted. The current formatting is default IntelliJ-IDE formatting, I have never seen a reason to differ from it. It makes it quite complicated to edit code, without having formatting possibility in my opinion.

File: test/tests.spec.js (hunk 1 above)
Comment (wiedehoeft) [reply test/tests.spec.js:66]: :(

**Comment by insidewhy** [comment]
@wiedehoeft If you want to reformat the code, then first open a PR moving the code over to `prettier`, and then after that's merged, you can make this change.

**Comment by wiedehoeft** [comment]
Ok, sorry. 
I thougt splitting it in several commits, could be enough.

**Pull request closed without merging**